clients/src/test/java/org/apache/kafka/common/ClusterTest.java

```diff
@@ -22,19 +22,35 @@
 import java.net.InetSocketAddress;
 import java.util.Arrays;
 import java.util.HashSet;
+import java.util.List;
 import java.util.Set;
 
+import static java.util.Arrays.asList;
 import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertThrows;
 
 public class ClusterTest {
 
+    private final static Node[] NODES = new Node[] {
```

Comment:
Maybe it's just me, but it seems surprising for `NODES[3]` to have an ID of 11.  Is there a reason for not just assigning them sequential IDs?

Reply:
I just copied that from the `StickyPartitionCacheTest` that @jolshan and you wrote. :) I thought that had been done that way to highlight the offline node.

Reply:
This may have just been a simple mistake on my part. It is in DefaultPartitionerTest as well.